### PR TITLE
[JenkinFile Integration]

### PR DESCRIPTION
**Pull request by arunsvce1991**

### CODE REVIEW

**Comment by CodeBreaker14** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by CodeBreaker14** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: undefined local variable or method `swiftlint` for #\<Danger::Dangerfile:0x00007fabf82eda00\>
```
 #  from Dangerfile:15
 #  -------------------------------------------
 #  
 >  swiftlint.lint_files
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: MoviesApp/Services/MoviesAPI.swift [hunk 1]
```diff
@@ -25,7 +24,7 @@ class MoviesAPIV1: MoviesAPI {
     
     convenience init(session: URLSession = URLSession(configuration:
         URLSessionConfiguration.default)) {
-        self.init(session: session, endPoint: "https://api.jsonbin.io/b/5e86bb0ef4dba3539aac2f2a")
+        self.init(session: session, endPoint: "http://api.jsonbin.io/b/5e86bb0ef4dba3539aac2f2a")
```
Comment (CodeBreaker14) [inline MoviesApp/Services/MoviesAPI.swift]: - :no_entry_sign: HTTPS should be favored over HTTP (`force_https`)

File: Dangerfile.swift [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (CodeBreaker14) [inline Dangerfile.swift:1]: - :no_entry_sign: File shouldn't start with whitespace: currently starts with 1 whitespace characters (`leading_whitespace`)

**Comment by CodeBreaker14** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesApp/MoviesCoordinator/MoviesCoordinator.swift#L19** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesApp/MoviesList/MoviesListViewController.swift#L11** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesApp/MoviesList/MoviesListViewModel.swift#L20** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesApp/RootClasses/AppDelegate.swift#L12** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesApp/Services/MoviesAPI.swift#L15** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesAppTests/MoviesCoordinatorTests.swift#L13** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  **MoviesAppTests/NetworkableTests.swift#L13** - Empty line required after class declarations (`space_after_class`)
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against 9238ec85f60c8a09d259b5bd7bd491df62113dfd
</p>

**Comment by arunsvce1991** [comment]
Closing PR

**Pull request closed without merging**